ci/build_container/build_recipes/nghttp2.sh

```diff
@@ -7,11 +7,40 @@ set -e
 # TODO(PiotrSikora): switch back to releases once v1.33.0 is out.
 VERSION=e5b3f9addd49bca27e2f99c5c65a564eb5c0cf6d  # 2018-06-09
 
-wget -O nghttp2-"$VERSION".tar.gz https://github.com/nghttp2/nghttp2/archive/"$VERSION".tar.gz
+curl https://github.com/nghttp2/nghttp2/archive/"$VERSION".tar.gz -sLo nghttp2-"$VERSION".tar.gz
 tar xf nghttp2-"$VERSION".tar.gz
 cd nghttp2-"$VERSION"
-autoreconf -i
-automake
-autoconf
-./configure --prefix="$THIRDPARTY_BUILD" --enable-shared=no --enable-lib-only
-make V=1 install
+
+# Allow nghttp2 to build as static lib on Windows
+cat > nghttp2_cmakelists.diff << 'EOF'
```

Comment:
Can you upstream this? Or at least add a TODO for this?

Reply:
We made a PR here: https://github.com/nghttp2/nghttp2/pull/1198, we can add a TODO to remove if /when it is merged